checker/src/org/checkerframework/checker/formatter/FormatterTreeUtil.java

```diff
@@ -386,12 +386,17 @@ public AnnotationMirror categoriesToFormatAnnotation(ConversionCategory[] args)
 
     /**
      * Takes a syntax tree element that represents a {@link Format} annotation,
-     * and returns its value.
+     * and returns its value. Returns null (instead of an empty array) if the
+     * {@code Format} annotation's value is empty.
      */
     public ConversionCategory[] formatAnnotationToCategories(AnnotationMirror anno) {
+        AnnotationValue av = AnnotationUtils.getElementValuesWithDefaults(anno).get(formatArgTypesElement);
+        if (av == null) {
+        	return null;
```

Comment:
It's a javac error to write `@Format()` because there is not default for value.  The only way that av is null is if the FormatChecker itself created it.